src/server/pfs/cmds/cmds.go

```diff
@@ -1017,6 +1021,59 @@ Any pachctl command that can take a commit, can take a branch name instead.`,
 	shell.RegisterCompletionFunc(deleteBranch, shell.BranchCompletion)
 	commands = append(commands, cmdutil.CreateAliases(deleteBranch, "delete branch", branches))
 
+	searchForFileInBranch := &cobra.Command{
+		Use:   "{{alias}} <repo>@<branch-or-commit>:<path/in/pfs> [flags]",
+		Short: "search for commits with reference to <filePath> within a branch starting from <repo@commitID>",
+		Long:  "search for commits with reference to <filePath> within a branch starting from <repo@commitID>",
+		Run: cmdutil.RunFixedArgs(1, func(args []string) error {
+			file, err := cmdutil.ParseFile(project, args[0])
+			if err != nil {
+				return err
+			}
+			commit := file.Commit
+			c, err := client.NewOnUserMachine("user")
+			if err != nil {
+				return err
+			}
+			defer c.Close()
+			resp, err := c.SearchForFileInBranch(&pfs.SearchForFileInBranchRequest{
+				Start:    commit,
+				FilePath: file.Path,
+				Timeout:  types.DurationProto(timeout),
+				Limit:    limit,
+			})
+			if err != nil {
+				return grpcutil.ScrubGRPC(err)
+			}
+			writer := tabwriter.NewWriter(os.Stdout, "FOUND IN COMMITS\n")
```

Comment:
+1 to `os.Stdout` and `tabwriter` instead of fmt.Println

Reply:
Ideally we would use the writer that cobra gives us (which enables better testing), but our `cmdutil` package throws that out so os.Stdout is fine.

Reply:
I figured this would be a better setup if we wanted to add more columns to the output later as well.